test/unit/test_items.py

```diff
@@ -47,3 +49,86 @@ def test_simple_2(self):
         self.assertIsNone(sco.get_secret_value())
         sco._exception = Exception("test")
         self.assertRaises(Exception, sco.get_secret_value)
+    def test_datetime_fix_is_refresh_needed(self):
+        sco = TestSecretCacheObject.TestObject(SecretCacheConfig(), None, None)
+
+        # Used to pass branching requirements (False is not None)
+        sco._next_retry_time = datetime.now(tz=timezone.utc)
+        sco._refresh_needed = False
+        sco._exception = not None
+
+        self.assertTrue(sco._is_refresh_needed())
+
+    def test_datetime_fix__refresh(self):
```

Comment:
nit: did you add an extra `_` for `__refresh` by accident?

Reply:
I did it on purpose because the function is called __refresh, but I can change that